pkg/utils/parse_arf_result_test.go

```diff
@@ -882,4 +882,77 @@ Server 3.fedora.pool.ntp.org`
 
 		})
 	})
+
+	Describe("Testing for correct content parsing", func() {
+		Describe("Testing new content used", func() {
+			Context("Valid XCCDF", func() {
+				if os.Getenv("DEFAULT_CONTENT_DS_FILE_PATH") != "" {
+					dsFilename = os.Getenv("DEFAULT_CONTENT_DS_FILE_PATH")
+				} else {
+					dsFilename = "RELACE_WITH_PATH_TO_DS_FILE"
+				}
+				if dsFilename != "RELACE_WITH_PATH_TO_DS_FILE" {
+					ds, err := os.Open(dsFilename)
+					Expect(err).NotTo(HaveOccurred())
+					dsDom, err := ParseContent(ds)
+					Expect(err).NotTo(HaveOccurred())
+					It("Should parse the XCCDF without errors", func() {
+						Expect(err).NotTo(HaveOccurred())
+					})
+					// printout dsDom xml structure
+					fmt.Println("dsDom structure")
+					printUniquePaths(dsDom, "", make(map[string]bool))
```

Comment:
Is `printUniquePaths()` here just for us to visualize the difference in parsing?

Reply:
correct